apps/marketplace/pages/AskQuestionPage.js

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Form, actions } from 'react-redux-form'
+import { withRouter } from 'react-router-dom'
+import DocumentTitle from 'react-document-title'
+import AUpageAlert from '@gov.au/page-alerts/lib/js/react.js'
+import AUheading from '@gov.au/headings/lib/js/react.js'
+import format from 'date-fns/format'
+import { loadPublicBrief, submitSupplierQuestion } from 'marketplace/actions/briefActions'
+import { required } from 'marketplace/components/validators'
+import Textarea from 'shared/form/Textarea'
+import { rootPath } from 'marketplace/routes'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+
+const model = 'askAQuestionForm'
+
+const getOpportunityUrl = brief =>
+  brief.lot === 'rfx' || brief.lot === 'atm' || brief.lot === 'specialist'
+    ? `${rootPath}/digital-marketplace/opportunities/${brief.id}`
+    : `/digital-marketplace/opportunities/${brief.id}`
+
+class AskQuestionPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: false,
+      errorMessage: '',
+      saved: false
+    }
+  }
+
+  componentWillMount() {
+    this.setState({
+      loading: true
+    })
+    const briefId = this.props.match.params.briefId
+    if (briefId.length > 0) {
```

Comment:
Do we want the check on length or the value of `briefId`?

Reply:
updated